aries_vcx/src/messages/connection/did_doc.rs

```diff
@@ -272,6 +272,7 @@ impl DidDoc {
     }
 }
 
+// TODO: Make into TryFrom
```

Comment:
can you create Github issue for this? I assume it would address the usage of `unwrap_or_else`, so we don't fail silently, right?

Reply:
That's the intent, correct.

Reply:
Issue created: https://github.com/hyperledger/aries-vcx/issues/439